desktop_version/src/Render.cpp

```diff
@@ -2158,6 +2158,91 @@ static const char* interact_prompt(
     return buffer;
 }
 
+static void mode_indicator_text(const int alpha)
+{
+    const int flags = PR_BRIGHTNESS(alpha) | PR_BOR;
+    const int r = 220 - help.glow;
+    const int g = 220 - help.glow;
+    const int b = 255 - help.glow/2;
+    const int x = 5;
+    const int spacing = font::height(flags) + 2;
+    int y = 5;
+    if (game.advancetext)
+    {
+        /* Prevent clashing */
+        y += 15;
+    }
+
+    /* FIXME: Some strings have not yet been translated. In order to not have
+     * English text in other languages, they are substituted with existing
+     * ones. Remove all substitute text when they're fully translated. */
+
+    if (map.invincibility)
+    {
+        const char* english = "Invincibility mode enabled";
+        const char* text = loc::gettext(english);
+        if (loc::lang != "en" && SDL_strcmp(english, text) == 0)
+        {
+            /* Substitute text */
+            text = loc::gettext("Invincibility");
+        }
+        font::print(flags, x, y, text, r, g, b);
+        y += spacing;
+    }
+
+    enum GlitchrunnerMode mode = GlitchrunnerMode_get();
+    if (mode != GlitchrunnerNone)
+    {
+        char buffer[SCREEN_WIDTH_CHARS + 1];
+        const char* english = "Glitchrunner mode enabled ({version})";
+        const char* text = loc::gettext(english);
+        if (loc::lang != "en" && SDL_strcmp(english, text) == 0)
+        {
+            /* Substitute text */
+            SDL_strlcpy(buffer, loc::gettext("Glitchrunner Mode"), sizeof(buffer));
+        }
+        else
+        {
+            const char* mode_string = loc::gettext(GlitchrunnerMode_enum_to_string(mode));
+            vformat_buf(
+                buffer, sizeof(buffer),
+                loc::gettext("Glitchrunner mode is {version}"),
+                "version:str", mode_string
+            );
```

Comment:
Is this format string not suppposed to be `text` instead?

Reply:
lol whoops